promscale/installation/index.md

```diff
@@ -9,8 +9,12 @@ related_pages:
   - /promscale/:currentVersion:/send-data/
 ---
 
+import PromscaleDeprecation from "versionContent/_partials/_deprecated-promscale.mdx";
+
 # Install Promscale
 
+<PromscaleDeprecation />
+
```

Comment:
@Loquacity could you remove lines 22-32 that promote Promscale with Timescale Cloud?

Reply:
Hi @ramonguiu !!! There is a related PR here: https://github.com/timescale/docs/pull/1999/files